src/components/sam-table/sam-table.tsx

```diff
@@ -1,10 +1,10 @@
-import React, { useEffect, useMemo, useState } from "react";
+import React from "react";
 import styles from './sam-table.module.css'
 import { Alignment, OrderDirection, Table } from "../table/table";
-import { formatPercentage, formatSolAmount, lamportsToSol } from "src/format";
+import { formatPercentage, formatSolAmount } from "src/format";
 import { Metric } from "../metric/metric";
 import { AuctionResult } from "@marinade.finance/ds-sam-sdk";
-import { selectBid, selectBondSize, selectCommission, selectEffectiveBid, selectConstraintText, selectMarinadeTargetStake, selectMaxAPY, selectMevCommission, selectMndeDistributedStake, selectMndeTargetStake, selectSamDistributedStake, selectSamTargetStake, selectVoteAccount, selectWinningAPY } from "src/services/sam";
```

Comment:
please, also remove the no longer unused methods

Reply:
already removed. those are still used